dist/plotly-strict.js

```diff
@@ -0,0 +1,6 @@
+/**
+* plotly.js (strict) v2.0.0
+* Copyright 2012-2021, Plotly, Inc.
+* All rights reserved.
+* Licensed under the MIT license
+*/
```

Comment:
Is this supposed to have content? Or is this just a stub until we make a release?

Reply:
Just a placeholder so that tests pass for now and to make sure that they would be present in the automatic commit once we run npm version major. It would be replaced by content at that time.